packages/salesforcedx-vscode-core/package.nls.json

```diff
@@ -74,5 +74,5 @@
   "force_function_start": "SFDX: Start Function",
   "force_function_invoke": "SFDX: Invoke Function",
   "force_function_stop": "SFDX: Stop Function",
-  "experimental_templates_library_description": "Use Templates Library for Templates CLI commands"
```

Comment:
Are we removing this flag?

Reply:
This wasn't my change. I saw this defined twice. Once in line 73. Can you confirm I'm doing this right.

Reply:
Yes let's remove this. maybe it was a merge conflict